File: 110algoritmos/exercicios010/ex001-Python/ex010.py
#               CONVERSOR DE MOEDAS
#Crie um programa que leia quanto dinheiro a pessoa tem na carteira e mostre
# quantos Dólares ela pode comprar.
# Considera $1,00 = 4.61
real = float(input("\nQuantos reais você tem na carteira? R$ "))
dollar = real / 4.61
print(f"\nCom R$ {real:.2f} reis você pode comprar $ {dollar:.2f} doláres.\n")

File: 110algoritmos/exercicios010/ex001-Elixir/ex010.exs
r = IO.gets("Quantos reais você tem na carteira? R$ ")
{reais, _} = Float.parse(r)
dollar = reais/4.61

IO.puts("\nCom R$ #{reais} reais voce pode comprar $ #{Float.round(dollar)} dollares.\n=-=-=-=-=-=")
